client/verta/verta/operations/monitoring/profiler.py

```diff
@@ -45,25 +123,70 @@ def _profile_column(self, df, column):
         return (column + "_missing", DiscreteHistogram(["present", "missing"], [total - missing, missing]))
 
 
+# TODO: Rename to CategoricalHistogramProfiler?
 class BinaryHistogramProfiler(Profiler):
+    """Produces discrete histograms counting values per unique value in columns.
+
+    Produces histogram columns for each unique value in the profiled column and
+    counts the number of values for these keys. This should not be used for
+    columns containing continuously measured values but should be used when the
+    set of possible values in a column is small, e.g. boolean valued columns.
+
+    Attributes
+    ----------
+    columns : :obj:`list` of :obj:`str`
+        The list of data frame columns which will be profiled.
+    """
+
     def __init__(self, columns):
+        """Initialize a BinaryHistogramProfiler for the specified columns.
+
+        Parameters
+        ----------
+        columns : :obj:`list` of :obj:`str`
+            The list of data frame columns which will be profiled.
+        """
         super(BinaryHistogramProfiler, self).__init__(columns)
 
-    def _profile_column(self, df, column):
+    def profile_column(self, df, column):
+        """Profile a data frame column, returning a name and DiscreteHistogram
+
+        Parameters
+        ----------
+        df : pandas.core.frame.DataFrame
+            A data frame to profile.
+        column : string
+            The column in the data frame to profile.
+
+        Returns
+        -------
+        dict
+            A dictionary from summary data names to DiscreteHistogram instances.
+        """
         content = df[column].value_counts()
         keys = list(content.keys())
         values = [content[k] for k in keys]
         values = [v.item() for v in values]
         return (column + "_histogram", DiscreteHistogram(values, keys))
 
 
+# TODO: Consider design/interface for different bins
 class ContinuousHistogramProfiler(Profiler):
+    """Produces float histograms counting values according to bin intervals.
+
+    Attributes
+    ----------
+    columns : :obj:`list` of :obj:`str`
```

Comment:
From my testing, the `:obj:` directive doesn't actually do anything. _But_ we can get the link to the std docs with https://github.com/VertaAI/modeldb/pull/2142.
```suggestion
    columns : list of str
```